packages/sdk/src/__tests__/schema-output.test.ts

```
@@ -0,0 +1,199 @@
+
+import { findSchemaMismatchReason, loadMinimalJsonSchema } from "../schema-output.js";
+
+describe("schema-output", () => {
+  it("loads a minimal JSON schema from disk", async () => {
+    const root = await mkdtemp(join(tmpdir(), "obora-schema-module-"));
+    await mkdir(join(root, "artifacts"), { recursive: true });
+    const schemaPath = join(root, "artifacts", "result.schema.json");
+    await writeFile(
+      schemaPath,
+      JSON.stringify({ type: "object", required: ["score"] }, null, 2),
+      "utf8",
```

ESLint

[error] Delete `,`

(prettier/prettier)